### DISCUSSION TITLE
"FATAL: Tenant or user not found" when using jdbc pooling string

### DISCUSSION THREAD

**Discussion by h4zm1** [supabase/supabase#20596] [Questions]
So am using spring boot and connecting to supabase directly like this in application.properties
```
spring.jpa.database=${JPA_DATABASE}
spring.datasource.username= postgres
spring.datasource.password= ${DATASOURCE_PASSWORD}
spring.datasource.url= jdbc:postgresql://db.lgmgbbhainyxbowxajqs.supabase.co:5432/postgres
spring.datasource.driver-class-name=${DATASOURCE_DRIVER_CLASS_NAME}
```

And it's working fine, but now when I wanted to switch to a pool connection I used this
```
spring.jpa.database=${JPA_DATABASE}
spring.datasource.username= postgres.lgmgbbhainyxbowxajqs
spring.datasource.password= ${DATASOURCE_PASSWORD}
spring.datasource.url=jdbc:postgresql://aws-0-ap-northeast-1.pooler.supabase.com:6543/postgres
spring.datasource.driver-class-name=${DATASOURCE_DRIVER_CLASS_NAME}
```
I ran it and got  FATAL: Tenant or user not found
![image](https://github.com/supabase/supabase/assets/24853128/85aa5828-c9e8-4733-b87a-39f3e205d7fc)

So not sure where the problem is
(am new to pooling connections btw)

**Comment by assembly-deepali** [comment]
@h4zm1 wondering if you found a solution! Running into the same issue right now.

**Reply by h4zm1** [reply]
Yeah I did, I totally forgot about this post or I would've come back and written what I found, anw, this's my current application.properties and it's working with the pooling string
```spring.jpa.database=postgresql
spring.datasource.username=postgres.projectid??
spring.datasource.password=yourpassword
spring.datasource.url=jdbc:postgresql://aws-0-eu-central-1.pooler.supabase.com:6543/postgres?prepareThreshold=0
spring.datasource.driver-class-name=org.postgresql.Driver
```
What am referring to as "projectid" is what you will see after /project/ in the url
